README.md

```diff
@@ -34,31 +34,35 @@ docker run -p 8080:8080 ghcr.io/medizininformatik-initiative/flare:2.1
-| FLARE_CACHE_MEM_EXPIRE          | PT48H                                 | The duration after which in-memory cache entries should expire in [ISO 8601 durations][1].       |
-| FLARE_CACHE_MEM_REFRESH         | PT24H                                 | The duration after which in-memory cache entries should be refreshed in [ISO 8601 durations][1]. |
-| FLARE_CACHE_DISK_PATH           | cache                                 | The name of the directory in which the on-disk cache should be written.                          |
-| FLARE_CACHE_DISK_EXPIRE         | P7D                                   | The duration after which on-disk cache entries should expire in [ISO 8601 durations][1].         |
-| FLARE_CACHE_DISK_THREADS        | 4                                     | The number of threads the disk cache should use for reading and writing entries.                 |
-| FLARE_MAPPING_MAPPING_FILE      | ontology/codex-term-code-mapping.json | The mappings to use.                                                                             |
-| FLARE_MAPPING_CONCEPT_TREE_FILE | ontology/codex-code-tree.json         | The code tree to use.                                                                            |
-| SERVER_PORT                     | 8080                                  | The port at which Flare provides its REST API.                                                   |
-| JAVA_TOOL_OPTIONS               | -Xmx4g                                | JVM options \(Docker only\)                                                                      |
-| LOG_LEVEL                       | info                                  | one of trace, debug, info, warn or error                                                         |
+| Name                              | Default                               | Description                                                                                      |
+|:----------------------------------|:--------------------------------------|:-------------------------------------------------------------------------------------------------|
+| FLARE_FHIR_SERVER                 | http://localhost:8082/fhir            | The base URL of the FHIR server to use.                                                          |
+| FLARE_FHIR_USER                   |                                       | The username to use for HTTP Basic Authentication.                                               |
+| FLARE_FHIR_PASSWORD               |                                       | The password to use for HTTP Basic Authentication.                                               |
+| FLARE_FHIR_OAUTH_TOKEN_URI        |                                       | The token URI of the OpenID Connect provider.                                                    |
```

Comment:
The token URI should be part of the OpenID Connect config you can load from `.well-known/openid-configuration`.  IHMO it would be simpler to just request the URI of the OpenID Connect provider as Blaze does it.